services/ui-src/src/views/StateHome/index.tsx

```diff
@@ -177,8 +177,8 @@ export const StateHome = () => {
     return <QMR.LoadingWave />;
   }
 
-  const filteredSpas = SPA.filter((s) => s.state === state);
-  const spaIds = filteredSpas.map((s) => s.id);
+  const filteredSpas = SPA[year!].filter((s: any) => s.state === state);
+  const spaIds = filteredSpas.map((s: any) => s.id);
```

Comment:
```suggestion
  const filteredSpas = SPA[year!].filter((s: SPAi) => s.state === state);
  const spaIds = filteredSpas.map((s: SPAi) => s.id);
```
In all of the instances where we're applying `any`, could we use the SPAi type? If we specify the type in `spaLib.ts` we may actually to just remove the type.